lib/matplotlib/widgets.py

```diff
@@ -159,12 +156,11 @@ def __init__(self, ax, label, image=None,
             The `~.axes.Axes` instance the button will be placed into.
 
         label : str
-            The button text. Accepts string.
+            The button text.
 
-        image : array, mpl image, Pillow Image
```

Comment:
"mpl images" are not a thing anymore since https://github.com/matplotlib/matplotlib/pull/5718/files#diff-327a40694cf498b1dd5e3f2ec0eed955.